api/src/main/java/io/terrakube/api/plugin/scheduler/job/tcl/executor/ephemeral/EphemeralExecutorService.java

```diff
@@ -39,7 +39,7 @@ public class EphemeralExecutorService {
     EphemeralConfiguration ephemeralConfiguration;
 
     public void send(Job job, ExecutorContext executorContext) throws ExecutionException {
-        final String jobName = "job-" + job.getId();
+        final String jobName = String.format("job-%s-%s", job.getId(), System.currentTimeMillis());
```

Comment:
I initially thought to use the step number for disambiguation, but it seems we don't have that here. The executor context step is the UUID which is not easy to find out, so not much info value.